.github/workflows/reusable-ci.yaml

```diff
@@ -0,0 +1,70 @@
+name: Reusable CI
+
+on:
+  workflow_call:
+
+jobs:
+  reusable-ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        otp-version: [24.1.2]
+        elixir-version: [1.12.3]
+
+    services:
+      db:
+        image: postgres:12
+        env:
+          POSTGRES_DB: elixir_boilerplate_test
+          POSTGRES_PASSWORD: boilerplate
+        ports: ["5432:5432"]
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    env:
+      MIX_ENV: test
+      DATABASE_URL: postgres://postgres:boilerplate@localhost/elixir_boilerplate_test
```

Comment:
The only difference is that the database used in the CI workflow will be named `elixir_boilerplate_test` (instead of `<project name>_test`). Same thing for the databse user credentials.

I could convert them to [_inputs_](https://docs.github.com/en/actions/learn-github-actions/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow) but I think it’s not a big deal, right?

Reply:
We could name it something more generic like `database_test`. If we get an error in our project with the mention of "elixir-boilerplate" it could be confusing.